test/select.spec.js

```diff
@@ -2902,8 +2902,21 @@ describe('ui-select tests', function () {
       clickMatch(el);
       triggerPaste(el.find('input'), 'tag1');
 
-      expect(scope.$broadcast).toHaveBeenCalledWith('uis:select', 'tag1');
-    })
+      expect(scope.$broadcast).not.toHaveBeenCalledWith('uis:select', 'tag1');
+      expect(scope.$broadcast).toHaveBeenCalledWith('uis:select-multiple', ['tag1']);
+    });
+
+    it('should broadcast pasted items once regardless of paste size', function() {
+      scope.pasteFunc = function(str) { return str.split(",") };
+      spyOn(scope, '$broadcast');
+
+      var el = createUiSelectMultiple({tagging: true, paste: "pasteFunc", taggingTokens: ","});
+      clickMatch(el);
+      triggerPaste(el.find('input'), 'tag1,tag2,tag3');
+
+      expect(scope.$broadcast).not.toHaveBeenCalledWith('uis:select', 'tag1');
+      expect(scope.$broadcast).toHaveBeenCalledWith('uis:select-multiple', ['tag1', 'tag2', 'tag3']);
+    });
 
     it('should not broadcast if paste function does not return pasted items', function() {
```

Comment:
Probably want to update this test too to make sure it doesn't broadcast `uis:select-multiple` instead here:

https://github.com/looker/ui-select/blob/9b816c11cc117219e47756630af5ee9cddf0206e/test/select.spec.js#L2929

Reply:
Oh yeah, you're right. Otherwise this will sneakily pass.